lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -109,9 +109,9 @@ def define
 
       #Q6
       multiple_choice :single_couple_lone_parent? do
-        option :single
-        option :couple
-        option :parent
+        query.rates.keys.each do |rate|
+          option rate
+        end
```

Comment:
I'm not completely sure about this change. It feels like we might be overloading the data in `rates` by using it to return the weekly benefit caps and to list the options to the `single_couple_lone_parent?`. How would it look if the question text for each of these options was also moved to the YAML file (as you've done with the benefits in 3ac69aaed833650a6b120de85d013fbd2a84b614)?